docs/source/rpc.rst

```diff
@@ -0,0 +1,42 @@
+.. role:: hidden
+    :class: hidden-section
+
+Distributed RPC Framework
+=====================================================
+
+The distributed RPC framework provides mechanisms for multi-machine model training through a set of primitives to allow for remote communication, and a higher-level API to automatically differentiate models split across several machines.
+
+RPC and RRef Framework
+====================================================
+
+Basics
+------
+
+Before using RPC and distributed autograd primitives, initialization must take place. First, a backend over which RPCs can be sent over must be initialized. The default (and currently, only available) implementation is the `ProcessGroup` backend, and must be initialized with `torch.distributed.init_process_group` before using other function. See the `documentation for torch.distributed <https://pytorch.org/docs/stable/distributed.html>`_ for additional details. Next, the local RPC agent can be initialized, after which the process will be able to send and receive RPCs from all other connected processes.
+
+.. automodule:: torch.distributed.rpc
+    :members:
+
+RRef
```

Comment:
Looks like the RRef class appears twice? 
![Screen Shot 2019-11-11 at 12 41 58 PM](https://user-images.githubusercontent.com/9958665/68619650-b9ef1580-0480-11ea-9838-58f7b43dd534.png)

Reply:
This is fixed, I had to manually specify `rpc.init_model_parallel` as an autofunction to get the desired order.

Reply:
Looks like the `RRef` class doesn't show up at all now.
![Screen Shot 2019-11-11 at 7 22 09 PM](https://user-images.githubusercontent.com/9958665/68639194-a5c70a80-04b8-11ea-90c8-a4eddd28dbea.png)